cmd/kube-apiserver/app/options/options.go

```diff
@@ -44,6 +44,7 @@ type APIServer struct {
 	AuthorizationMode          string
 	AuthorizationConfig        apiserver.AuthorizationConfig
 	BasicAuthFile              string
+	DefaultStorageMediaType    string
```

Comment:
Will do this in a follow up given priority.  There are other follow ups and @wojtek-t and I need to be enabling this now.